minimal_scheduling/CMakeLists.txt

```diff
@@ -26,11 +26,15 @@ ament_target_dependencies(minimal_scheduling_spin_thread rclcpp std_msgs)
 add_executable(minimal_scheduling_callback_group minimal_scheduling_callback_group.cpp)
 ament_target_dependencies(minimal_scheduling_callback_group rclcpp std_msgs)
 
+add_executable(minimal_scheduling_real_time_tutorial minimal_scheduling_real_time_tutorial.cpp)
```

Comment:
@JanStaschulat This executable might be used independently of the ROS 2 tutorial. I would change the name of the executable to something describing what this example is doing instead of describing where it is used.

Reply:
changed filename to `minimal_scheduling_real_time_executor`